### PR TITLE
Implementing flick action in ajio mobile

### PR DESCRIPTION
**Pull request by raghavgarg1996**
Implemented flick action gesture in ajio mobile.

### CODE REVIEW

File: src/main/java/com/znsio/teswiz/runner/Driver.java [hunk 1]
```diff
@@ -603,4 +603,24 @@ public void doubleTap(WebElement element) {
...
+    public void flick() {
+        AppiumDriver appiumDriver = (AppiumDriver) this.driver;
+        Dimension screenSize = driver.manage().window().getSize();
+
+        int startX = screenSize.width - 100;
+        int startY = screenSize.height / 2;
+        int endX = screenSize.width / 2;
+        int endY = screenSize.height / 2;
+
+        PointerInput finger = new PointerInput(PointerInput.Kind.TOUCH, "finger");
```
[the remark this answers is not in the document]
Comment (raghavgarg1996) [reply src/main/java/com/znsio/teswiz/runner/Driver.java:617]: In order to validate the flick - I am capturing the bound attribute before flick and after flick - and comparing that their values should not be same

For information: Adding loggers here as well

File: src/test/java/com/znsio/teswiz/screen/android/ajio/SearchScreenAndroid.java [hunk 2]
```diff
@@ -33,10 +36,23 @@ public int numberOfProductFound() {
...
+            driver.tapOnMiddleOfScreen();
+        }
         List<WebElement> list = driver.waitTillPresenceOfAllElements(byProductId);
         list.get(0).click();
+        return ProductScreen.get();
+    }
+
+    @Override
+    public boolean isProductListLoaded(String product) {
+        LOGGER.info(String.format("Verifying if %s list is loaded",product));
```
Comment (snehith07) [inline src/test/java/com/znsio/teswiz/screen/android/ajio/SearchScreenAndroid.java:51]: format the code properly. Opn + Cmd + L

File: src/test/java/com/znsio/teswiz/screen/android/ajio/SearchScreenAndroid.java [hunk 3]
```diff
@@ -33,10 +36,23 @@ public int numberOfProductFound() {
     }
 
     @Override
-    public void selectProduct() {
-        LOGGER.info("selection of Product in the result page");
-        driver.waitTillElementIsPresent(By.id("com.ril.ajio:id/layout_category_container")).click();
+    public ProductScreen selectProduct() {
+        LOGGER.info("Selection of Product in the result page");
+        if (!(driver.isElementPresent(byProductLayoutId))) {
```
Comment (snehith07) [inline src/test/java/com/znsio/teswiz/screen/android/ajio/SearchScreenAndroid.java:41]: do we need extra braces in if condition? 
if(!driver.isElementPresent(byProductLayoutId))

Comment (raghavgarg1996) [reply src/test/java/com/znsio/teswiz/screen/android/ajio/SearchScreenAndroid.java:41]: yes - when we land on the AJIO app on the first time - few sections got highlighted - and web elements are present below these elements

File: src/test/java/com/znsio/teswiz/screen/android/ajio/SearchScreenAndroid.java (hunk 2 above)
Comment (raghavgarg1996) [reply src/test/java/com/znsio/teswiz/screen/android/ajio/SearchScreenAndroid.java:51]: Done - missed it earlier

File: src/test/java/com/znsio/teswiz/screen/android/ajio/SearchScreenAndroid.java [hunk 4]
```diff
@@ -15,6 +16,8 @@ public class SearchScreenAndroid
     private static final Logger LOGGER = Logger.getLogger(SCREEN_NAME);
     private static final By byResultsId = By.id("com.ril.ajio:id/tv_count_plp_header_is");
     private static final By byProductId = By.id("com.ril.ajio:id/plp_row_product_iv");
+    private static final By byProductListTitleId = By.id("com.ril.ajio:id/toolbar_title_tv");
+    public static final By byProductLayoutId = By.id("com.ril.ajio:id/layout_category_container");
```
Comment (snehith07) [inline src/test/java/com/znsio/teswiz/screen/android/ajio/SearchScreenAndroid.java:20]: public variable? 
are you using this VAR in another class?

**Pull request merged**